src/main/scanLogic/scanRunners/iacScan.ts

```diff
@@ -18,13 +18,14 @@ export class IacRunner extends BinaryRunner {
         connectionManager: ConnectionManager,
         logManager: LogManager,
         binary?: Resource,
-        timeout: number = ScanUtils.ANALYZER_TIMEOUT_MILLISECS
+        timeout: number = ScanUtils.ANALYZER_TIMEOUT_MILLISECS,
+        verbose: boolean = false
```

Comment:
Instead of pushing the constructor to have numerous optional parameters, it's preferable to remove all optional parameters. By employing setters, we can build objects step by step and achieve better code organization and readability.